### PR TITLE
feat: add bottom nav in mobile

### LINKED ISSUE #43: Add bottom nav in mobile

### PR DESCRIPTION
**Pull request by ercwangwh**
## What does this PR do?

Fixes #43  (issue)

## Type of change

- [ ] New feature (non-breaking change which adds functionality)
- [x] Enhancement (non-breaking small changes to existing functionality)

## How should this be tested?

- [ ] Should nav correctly
- [ ] Should only showup above lg width
- [ ] Should not showup if users not login

### CODE REVIEW

File: src/components/Shared/BottomNav/index.tsx [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+      {currentProfile ? (
+        <div className="container px-5 mx-auto max-w-screen-xl">
+          <div className="flex relative justify-between items-center h-14 sm:h-16">
+            <NavItems />
+            <MessageIcon />
+            <NotificationIcon />
+          </div>
+        </div>
+      ) : (
+        <div />
```
Comment (shanpriyan) [inline src/components/Shared/BottomNav/index.tsx:76]:
```suggestion
        null
```

**Comment by bigint** [comment]
@ercwangwh I'm gonna look at this today, can you please resolve the conflict 🙏🏼

**Comment by bigint** [comment]
I think we need bottom nav for non logged in users as well 🙇🏼

**Comment by bigint** [comment]
Have we checked this on PWA? is this working fine in both iPhone and android?

AFAIK in iPhone the bottom bar will break 🙇🏼

**Comment by ercwangwh** [comment]
> 

Allowing unlogged users to use the bottom navigation is a good idea and I haven't experienced any crashes on both iPhone and Android. However, I think what I've done is a temporary solution and to improve the UX, we need to redesign the header and bottom nav for mobile. Currently, the header and bottom nav have some feature overlap, such as the explore feat. Some feature entries are too deep, such as the report bug feature.

**Comment by ercwangwh** [comment]
@bigint If currently we don't need a temporary bottom nav solution, I’ll close this PR. I’ll request a new one when lenster team or me has a better mobile UI design. 🙏

**Comment by bigint** [comment]
@ercwangwh yup, we will come with design asap and update in the issue 🙏🏼 

So closing this now, thanks for the work fren 🙏🏼

**Pull request closed without merging**